cfgmgr/teammgr.cpp

```diff
@@ -504,6 +504,52 @@ bool TeamMgr::removeLag(const string &alias)
     return true;
 }
 
+// Port-channel names are in the pattern of "PortChannel####"
+// 
+// The LACP key could be generated in 3 ways based on the value in config DB:
+//      1. "auto" - LACP key is extracted from the port-channel name and is set to be the number at the end of the port-channel name
```

Comment:
since you added new config db knob, please add this to the sonic yang model. https://github.com/Azure/sonic-buildimage/tree/master/src/sonic-yang-models/yang-models